dask/array/tests/test_routines.py

```diff
@@ -39,27 +51,53 @@ def test_array():
     ((4, 6, 8), (2, 3, 4)),
     ((4, 6, 8, 10), (2, 3, 4, 5)),
 ])
-def test_atleast_nd(funcname, num_arrs, shape, chunks):
+def test_atleast_nd_one_arg(funcname, shape, chunks):
     np_a = np.random.random(shape)
     da_a = da.from_array(np_a, chunks=chunks)
 
-    np_a_n = num_arrs * [np_a]
-    da_a_n = num_arrs * [da_a]
+    np_func = getattr(np, funcname)
+    da_func = getattr(da, funcname)
+
+    np_r = np_func(np_a)
+    da_r = da_func(da_a)
+
+    assert_eq(np_r, da_r)
+
+
+@pytest.mark.parametrize("funcname", [
+    "atleast_1d",
+    "atleast_2d",
+    "atleast_3d",
+])
+@pytest.mark.parametrize("shape1, shape2", list(
+    itertools.combinations_with_replacement(
```

Comment:
I didn't know about this function, looks useful.

Reply:
Yeah its kind of buried in `itertools`, but definitely handy for cases like this.